tests/multi_output_models.py

```diff
@@ -1,124 +1,101 @@
+from typing import Any, Dict
+
 import numpy as np
 
 from sklearn.base import clone
 from sklearn.pipeline import make_pipeline
 from sklearn.preprocessing import FunctionTransformer, OneHotEncoder, OrdinalEncoder
 from tensorflow.python.keras.losses import is_categorical_crossentropy
 
-from scikeras.utils.transformers import Ensure2DTransformer
+from scikeras.utils import Ensure2DTransformer, KerasClassifierTargetTransformer
 from scikeras.wrappers import BaseWrapper, KerasClassifier
 
 
-class MultiOuputClassifier(KerasClassifier):
+class MultiOutputKerasClassifierTargetTransformer(KerasClassifierTargetTransformer):
```

Comment:
:-1: I think this class is too complex to go in the tests; it's about 70 lines, which is far more than a "simple class," which I consider to be 10 lines or less. I would consider this a simple class:

I do one of the following:

* Put this code in `scikeras/`, not `tests/`.
* Reduce the complexity of this class (by reducing the LOC required for this class).

You could also remove the encoders and have a simple multiple IO class:

``` python
class MultiOutputClassifier(KerasClassifier):
    @property
    def target_encoder(self):
        return FunctionTransformer(
            transform=lambda X: [X[:, 0], X[:, 1], X[:, 2:]],
            inverse_transform=np.concatenate,
        )
```

Reply:
I think this could be simplified like you indicate, but it would require removing tests that use this class + `{"sparse_categorical_crossentropy", "categorical_crossentropy"}`. I think we should be fine testing only with `"binary_crossentropy"`. We can save this more complex example for a notebook. What do you think?

Reply:
Turns out there was a single test that needed editing 😄 . The implementation is now only ~50 LOC, and most of it is boilerplate/docstrings.

Reply:
Simplified even further, looking at 27 lines of actual code. I think that should be acceptable.